op-batcher/rpc/api.go

```diff
@@ -34,6 +38,36 @@ func GetAdminAPI(api *adminAPI) gethrpc.API {
 	}
 }
 
+func (a *adminAPI) UseDAType(
```

Comment:
I think this would be better if there was a getter + setter where the getter returns the whole config and the setter lets you set the whole config. This is to prevent the need to add more RPC endpoints for particular things. The tradeoff here is around a lot of the config being standard and there isn't a huge need to change it

Reply:
Ack - returning whole config in caused an import cycle